src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,32 @@
 package core.basesyntax;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
Since the formatter is created here and used multiple times, it should be made a constant field. This will make your code easier to understand and maintain.